jenkins-pipelines/nemesis/longevity-5gb-1h-ToggleLdapConfiguration-aws.jenkinsfile

```diff
@@ -7,6 +7,6 @@ longevityPipeline(
     backend: 'aws',
     region: 'eu-west-1',
     test_name: 'longevity_test.LongevityTest.test_custom_time',
-    test_config: '''["test-cases/nemesis/longevity-5gb-1h-ToggleLdapConfiguration.yaml", "configurations/ldap-authorization.yaml"]'''
```

Comment:
looks like you are breaking this one, and you have one more exception

Reply:
Fixed